spec/round_spec.rb

```diff
@@ -0,0 +1,197 @@
+require './lib/card'
+require './lib/deck'
+require './lib/round'
+require './lib/turn'
+
+RSpec.describe Round do
+  describe '#initialize' do
+    it 'creates an instance of Round' do
+      card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+      card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the serface of which planet?", "Mars", :STEM)
+      card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due North?", "North north west", :STEM)
+      cards = [card_1, card_2, card_3]
+      deck = Deck.new(cards)
+      round = Round.new(deck)
+
+      expect(round).to be_instance_of(Round)
+    end
+  end
+
+describe '@deck' do
+    it 'has a deck of cards' do
+      card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+      card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the serface of which planet?", "Mars", :STEM)
+      card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due North?", "North north west", :STEM)
+      cards = [card_1, card_2, card_3]
+      deck = Deck.new(cards)
+      round = Round.new(deck)
+
+      expect(round.deck).to be_instance_of(Deck)
+    end
+  end
+
+  describe '@turns' do
+    it 'begins with an empty array' do
+      card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+      card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the serface of which planet?", "Mars", :STEM)
+      card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due North?", "North north west", :STEM)
+      cards = [card_1, card_2, card_3]
+      deck = Deck.new(cards)
+      round = Round.new(deck)
+
+      expect(round.turns).to eq([])
+    end
+  end
+
+  describe '#current_card' do
```

Comment:
I really like your consistency when it comes to testing, your tests are very well organized. If i were joining your group as a contributing developer, I feel like I would have no trouble figuring out what you are testing for!